### PR TITLE
no_std addr2line

### PR DESCRIPTION
**Pull request by roblabla**
Fixes #98

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -37,7 +37,9 @@ debug = true
 debug = true
 
 [features]
-default = ["rustc-demangle", "cpp_demangle"]
+default = ["std", "rustc-demangle", "cpp_demangle"]
+std = ["gimli/std", "object/std", "cpp_demangle/std"]
+nightly = ["gimli/alloc", "fallible-iterator/alloc"]
```
Comment (philipc) [inline Cargo.toml]: Please call this `alloc`.

File: src/lib.rs [hunk 2]
```diff
@@ -35,9 +52,11 @@ extern crate rustc_demangle;
 extern crate smallvec;
 
 use std::cmp::Ordering;
-use std::borrow::Cow;
+use alloc::borrow::Cow;
 use std::u64;
-use std::rc::Rc;
+use alloc::rc::Rc;
+use alloc::vec::Vec;
+use alloc::string::{ToString, String};
```
Comment (philipc) [inline src/lib.rs]: Group the `alloc` imports together.

**Comment by roblabla** [comment]
Ugh, sorry, I tested on an old nightly (I'm stuck on an old nightly for the time being).

**Comment by roblabla** [comment]
Right so it requires an object release (for the goblin bump), and a stable_deref_trait release.

**Comment by roblabla** [comment]
stable_deref_trait got a release. We're only missing an object release now.

**Comment by philipc** [comment]
It needs a gimli release too. Also this still doesn't build. It might be simpler if I reopen #98 (which doesn't need any changes other than a rebase).

**Pull request closed without merging**